lib/room.cpp

```diff
@@ -1287,6 +1291,60 @@ bool isEchoEvent(const RoomEventPtr& le, const PendingEventItem& re)
     return le->contentJson() == re->contentJson();
 }
 
+bool Room::processCall(Room* room, const RoomEvent* event)
+{
+    if (!room->isCallSupported()) {
+        qCDebug(MAIN) << "Got call event in room with more then two"
+            << "members, Ignoring this event!";
+        return true;
+    }
+    emit callEvent(room, event);
+    return false;
```

Comment:
I'm afraid there's a misconception here. The return value of `processStateEvent()` indicates whether the display name of the room should be recalculated; since `processCall()` doesn't affect it in any way, it should return `false` in all branches.